content/applications/inventory_and_mrp/inventory/barcode/setup/hardware.rst

```diff
@@ -4,64 +4,68 @@ Set up your barcode scanner
+Follow this guide to choose and set up a barcode scanner compatible with Odoo's *Inventory* and
+*Barcode* apps.
+
+.. image:: hardware/barcode-scanner.png
+   :align: center
+   :alt: An image of an example barcode scanner.
+
+Scanner types
+=============
+
+To determine which barcode scanner is most compatible with Odoo and the business' needs, refer to
+the following section:
+
+- **USB scanners** are connected to a computer and are suitable for businesses that scan products at
+  a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
+  compatible with the keyboard layout of the computer.
+
+- **Bluetooth scanners** pair with a smartphone or tablet, making them an ideal cost-effective and
+  portable barcode scanner option. In this scenario, Odoo is installed on the smartphone, allowing
+  warehouse operators to handle operations and check stock directly through their mobile devices.
+
+- **Mobile computer scanners** are mobile devices with a built-in barcode scanner. Ensure the device
+  can run the Odoo mobile app properly, and recent models that use Android OS with the Google Chrome
+  browser or Windows OS with Internet Explorer Mobile should work. However, testing is crucial due
+  to the variety of available models and configurations.
+
+Configuration
+=============
+
+When setting up the barcode scanner, ensure the following configurations are correctly set to ensure
+the scanner can interpret barcodes properly in Odoo.
 
 Keyboard layout
 ---------------
 
-.. image:: hardware/hardware01.png
-    :align: center
-
-An USB barcode scanner needs to be configured to use the same keyboard
-layout as your operating system. Otherwise, your scanner won't translate
-characters correctly (replacing a 'A' with a 'Q' for example). Most
-scanners are configured by scanning the appropriate barcode in the user
-manual.
+When using a USB barcode scanner, match its keyboard layout with the operating system's layout for
+proper interpretation of characters. To configure the keyboard layout, scan the keyboard wedge
+barcode in the user manual for corresponding to the barcode scanner.
 
 Automatic carriage return
 -------------------------
 
-By default, Odoo has a 50 milliseconds delay between each successive
-scan (it helps avoid accidental double scanning). If you want to
-suppress this delay, you can configure your scanner to insert a carriage
-return at the end of each barcode. This is usually the default
-configuration and can be explicitly configured by scanning a specific
-barcode in the user manual ('CR suffix ON', 'Apply Enter for suffix',
-etc.).
+Odoo has a default 50-millisecond delay between scans to prevent accidental double scanning. To
+remove this delay, configure the scanner to add a carriage return at the end of each barcode.
+Typically, this is the default setting and can be explicitly set by scanning a specific barcode in
+the user manual, like 'CR suffix ON' or 'Apply Enter for suffix.'
```

Comment:
I think this section needs to be expanded. It leaves me with more questions than answers. What is a carriage return? Why would I want to use one instead of a delay? If the default for Odoo is a delay, but the default for the scanner is a carriage return, which one takes precedence? Etc.

Reply:
what do you think about this re-written section/.
> Odoo has a default 50-millisecond delay between scans to prevent accidental double scanning. To
synchronize with the barcode scanner, set it to include a *carriage return* (:dfn:`character like
the "Enter" key on a keyboard`) after each scan. Odoo interprets the carriage return as the end of
the barcode input; so Odoo accepts the scan and waits for the next one.

Typically, on the scanner, a carriage return is included by default. Ensure it is set by scanning a
specific barcode in the user manual, like 'CR suffix ON' or 'Apply Enter for suffix.'